auth/realm/jdbc/src/main/java/org/wildfly/security/auth/realm/jdbc/QueryBuilder.java

```diff
@@ -34,7 +37,16 @@
     private List<ColumnMapper> mappers = new ArrayList<>();
     private DataSource dataSource;
 
+    /**
+     *
+     * @param sql (must not be {@code null})
+     * @param parent (must not be {@code null})
+     *
+     */
     QueryBuilder(String sql, JdbcSecurityRealmBuilder parent) {
+        assertNotNull(sql, "sql should not be Null");
```

Comment:
Because we're using the `Assert` class from the `org.wildfly.common.Assert` package, this should be (the first param is the name of variable and the second param is the variable we want to check):

Assert.checkNotNullParam("sql", sql);
Assert.checkNotNullParam("parent", parent);